ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -455,11 +455,16 @@ extension OnboardingCoordinator: ESIMOnBoardingDelegate {
 }
 
 extension OnboardingCoordinator: ESIMInstructionsDelegate {
-    func completedInstructions() {
+    func completedInstructions(_ controller: ESIMInstructionsViewController) {
         localContext.hasSeenESIMInstructions = true
         
+        let spinner = controller.showSpinner()
```

Comment:
By calling`controller.showSpinner()`, it will prevent user from clicking the continue twice, right ?

Reply:
Indeed. Showing the spinner blocks all of the UI so you can't tap on anything.